lib/utils/orderSchemasByDependency.js

```diff
@@ -130,9 +130,10 @@ function processSchema(schemaAndInformation, independent, attributePlaceHolder,
       schemaAndInformation.dependencies.add(
         getXMLAttributeByName(importElement, attributePlaceHolder, 'namespace'));
       assignNamespacesToSchema(schemaAndInformation, namespacesFromWSDL, attributePlaceHolder);
-      if (toProcess && parentTargetNamespace !== importedName) {
+      if (toProcess && processedSchemas.find((schema) => { return importedName === schema; }) === undefined) {
```

Comment:
Maybe we could take it: 'processedSchemas.find((schema) => { return importedName === schema; }) === undefined' out to a variable.

let isNotProcessedSchema = !processedSchemas.find((schema) => { return importedName === schema; })

Reply:
Yes, i think creating a variable reflects better the intention